packages/shorebird_cli/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: shorebird_cli
 description: Command-line tool to interact with Shorebird's services.
-version: 1.5.5
+version: 1.6.0
```

Comment:
We'll need to bump and redeploy the backend if we're bumping to 1.6.0

Reply:
Done